src/utilities/boleto/index.ts

```diff
@@ -134,3 +140,40 @@ export function format(boleto: string) {
       return result;
     }, '');
 }
+
+export function getValueInCents(digitableLine: string): number {
+  if (!digitableLine || !isValid(digitableLine)) return 0;
+
+  const digits = onlyNumbers(digitableLine);
+
+  const valueStartIndex = digits.length - 10;
+
+  return Number(digits.substr(valueStartIndex));
+}
+
+export function getExpirationDate(digitableLine: string): Date | null {
+  if (!digitableLine || !isValid(digitableLine)) return null;
+
+  const firstDay = new Date(1997, 9, 7);
```

Comment:
it can be a constant outside of this function called `BANCO_CENTRAL_BASE_DATE` (or something that you think is better)